contracts/test/integration/IntegrationTestPriceBoundPSM.t.sol

```diff
@@ -138,20 +148,22 @@ contract IntegrationTestPriceBoundPSMTest is DSTest {
 
     /// @notice PSM is set up correctly and view functions are working
     function testGetMintAmountOut() public {
-        uint256 amountFeiIn = 100;
-        assertEq(psm.getMintAmountOut(amountFeiIn), 98);
-    }
+        uint256 currentPegPrice = oracle.getCurrentOraclePrice();
 
-    /// @notice PSM is set up correctly and view functions are working
-    function testGetRedeemAmountOutAfterTime() public {
-        uint256 amountVoltIn = 98;
-        uint256 expectedAmountStableOut = 99;
+        uint256 fee = (mintAmount * psm.mintFeeBasisPoints()) /
+            Constants.BASIS_POINTS_GRANULARITY;
+
+        uint256 amountOut = ((mintAmount.mul(1e18) / currentPegPrice)) - fee;
 
-        assertEq(psm.getRedeemAmountOut(amountVoltIn), expectedAmountStableOut);
+        assertApproxEq(
+            psm.getMintAmountOut(mintAmount).toInt256(),
+            amountOut.toInt256(),
+            1
```

Comment:
theoretically the math we did here should match up exactly with the numbers the PSM arrives at, so basis point deviation should be 0, so we should be able to use assertEq